coldfront/core/utils/management/commands/update_state_and_extra_fields.py

```diff
@@ -0,0 +1,88 @@
+'''
```

Comment:
(Future work might include actually filling in `request.mou_file` with a reference to the file in Google Drive, but that's not super important right now, as long as existing request pages don't break as a result.)